bin/checkver.ps1

```diff
@@ -74,6 +74,7 @@ param(
 
 $Dir = Resolve-Path $Dir
 $Search = $App
+$GitHubToken = $env:SCOOP_CHECKVER_TOKEN, (get_config 'checkver-token') | Where-Object { -not [String]::IsNullOrEmpty($_) } | Select-Object -First 1
```

Comment:
```suggestion
$GitHubToken = $env:SCOOP_CHECKVER_TOKEN, (get_config 'checkver-token') | Where-Object -Property Length -Value 0 -GT | Select-Object -First 1
```

Reply:
A small tweak that may be faster.

Other part LGTM

Reply:
Done